seqsplit/seqsplit.go

```diff
@@ -0,0 +1,139 @@
+// Copyright ©2017 The bíogo Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Split sequence that is above the min cut-off length to generate fragments
+// such that each fragment falls in the range: window < fragment < 2*window
+// (i.e. 5 to 10 kb).  To achieve this, split each contig into window-sized
```

Comment:
To achieve this each contig is split into window-sized... and the remainder is added to the last window...